### PR TITLE
Small fixes in attribute documentation.

### PR DESCRIPTION
**Pull request by Zimmi48**
In particular, move the footnotes back to the foot of the chapter.

**Kind:** documentation fix

### CODE REVIEW

**Comment by Zimmi48** [comment]
ping @coq/doc-maintainers

File: doc/sphinx/language/gallina-specification-language.rst [hunk 1]
```diff
@@ -1459,3 +1452,12 @@ Here are a few examples:
         Proof.
           now foo.
         Abort.
+
+.. [1]
+   This is similar to the expression “*entry* :math:`\{` sep *entry*
+   :math:`\}`” in standard BNF, or “*entry* :math:`(` sep *entry*
+   :math:`)`\ \*” in the syntax of regular expressions.
```
Comment (cpitclaudel) [inline doc/sphinx/language/gallina-specification-language.rst]: Why do we use math rather than code for bnf?

**Review by cpitclaudel — APPROVED** [review]

**Comment by Zimmi48** [comment]
Closing / reopening to test @coqbot update.

**Comment by cpitclaudel** [comment]
Can you set a milestone on this PR? The script is complaining about that :innocent:

**Comment by Zimmi48** [comment]
Done.

File: doc/sphinx/language/gallina-specification-language.rst (hunk 1 above)
Comment (Zimmi48) [reply doc/sphinx/language/gallina-specification-language.rst]: I had missed this comment, and indeed I can change this if you want.

Comment (cpitclaudel) [reply doc/sphinx/language/gallina-specification-language.rst]: I'd rather merge this and address the rest in a separate PR :)

**Pull request merged**